packages/edge-login-ui-rn/src/native/connectors/componentConnectors/HeaderRecoverPasswordLogin.js

```diff
@@ -12,7 +12,6 @@ export const mapStateToProps = (state: State) => {
     showBackButton: currentScene.back,
     showSkipButton: currentScene.skip,
     title: currentScene.title,
-    subTitle: currentScene.subTitle,
```

Comment:
subtitle is a required string.  please actually fix the content in the Workflow reducer. Look at line 87 and start to change the subtitles to empty strings.